main.py

```diff
@@ -1,7 +1,36 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.bulk_create(
+        Genre(name=genre)
+        for genre in ["Western", "Action", "Dramma"]
```

Comment:
Better to save ["Western", "Action", "Dramma"] in variable, but for this task, it not necessary